swatch-contracts/build.gradle

```diff
@@ -71,6 +61,7 @@ dependencies {
     testImplementation("io.quarkus:quarkus-test-kafka-companion")
     testImplementation 'org.testcontainers:postgresql'
     testImplementation project(':swatch-common-testcontainers')
+    testImplementation project(':swatch-database') // To allow test creation of database tables
```

Comment:
Adding this dependency is also causing failures in tests for Quarkus projects because it overwrites the DATASOURCE and forcing to use always a postgres database instead of the h2 in-memory database.